### DISCUSSION TITLE
How can I upgrade included mpv to newer version?

### DISCUSSION THREAD

**Discussion by rtabulov** [mpvnet-player/mpv.net#603] [Q&A] [answered]
The newer mpv version had some bug fixes since August, how can I upgrade it?

**Comment by stax76** [comment, marked as answer]
You can find libmpv builds here:

https://github.com/zhongfly/mpv-winbuild

Downloads are on the release page:

https://github.com/zhongfly/mpv-winbuild/releases

The file you want is:

[mpv-dev-x86_64-v3-20231207-git-f886eb5.7z](https://github.com/zhongfly/mpv-winbuild/releases/download/2023-12-07-f886eb5/mpv-dev-x86_64-v3-20231207-git-f886eb5.7z)

`dev` means libmpv, what you want.

`x86_64` means x64, what you want.

`v3` means modern CPU optimization, works only on modern CPUs, there is also a download for old CPU without `v3`.

`20231207` the date.

`git-f886eb5` the id/name of the last git commit.

In the archive you want the file `libmpv-2.dll`, that is the libmpv binary, just replace this file.
